src/utils.ts

```diff
@@ -92,6 +94,7 @@ export enum OracleCurrencyPair {
   USDXOF = 'USDXOF',
   EUROCXOF = 'EUROCXOF',
   EUROCUSDT = 'EUROCUSDT',
+  KESUSD = 'KESUSD',
```

Comment:
Does this also need to be added to the core currency pair array?

Reply:
`CELOKES` should be added, updated